content/60.methods.md

```diff
@@ -1,50 +1,101 @@
-## Application of an Open Publishing Framework to an Emerging Public health Crisis
+## An Open-Publishing Response to an Infodemic
 
-### Abstract
+### ABSTRACT
+
+In an effort to keep pace as new information about COVID-19 and SARS-CoV-2 becomes available, this project is an open, collaborative effort that invited contributions from the scientific community broadly, similar to previous efforts to develop collaborative reviews [@doi:10.1098/rsif.2017.0387; @url:https://greenelab.github.io/deep-review].
+
+
+### CCS CONCEPTS
```

Comment:
Does this refer to https://dl.acm.org/ccs?

Reply:
It was listed as a required section of the manuscript, but that's also what I found when I looked it up! @mprobson any thoughts?

Reply:
> Does this refer to https://dl.acm.org/ccs?

Yes. We'll select topics and it will auto-generate code to put here.